docs/Ways-of-Working/Guides/Backlog-Refinement-Session-Guidelines.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Guide to Backlog Refinement Sessions
+authors:
+  - Amy Gray
+reviewed:
+reviewer:
+next-review: 2024-06-01
+---
+
+## Backlog Refinement
+
+The purpose of this meeting is for the team to review the prioritised items in the backlog before accepting them. We will make necessary adjustments for clarity and estimate the effort it will take to complete each Backlog Item.
```

Comment:
```suggestion
The purpose of this meeting is for the team to review the prioritised items in the backlog before accepting them. We will make necessary adjustments for clarity and estimate the effort it will take to complete each Backlog Item (BLI).
```

Might be worth adding BLI just to make it really explicit that BLI means Backlog Item?